### DISCUSSION TITLE
More system data available, please

### DISCUSSION THREAD

**Discussion by Pinpineapple86** [rocksdanister/lively#2449] [Ideas]
Greetings development team

As a suggestion, it would be great to have more system data available to be displayed like CPU temperature, fan status, WiFi network speed or even SSD temperature. There is a lot of system information that it is a shame not to be able to have it in a wallpaper.

Thanks for taking your time to read this.

**Comment by rocksdanister** [comment]
Some of those data like temperature require running the program elevated so had to be avoided.

I did look into some third party libraries but there is a risk of av program flagging them due to the way it works.
